auction-server/src/kernel/analytics_db.rs

```diff
@@ -0,0 +1,53 @@
+use {
+    clickhouse::Row,
+    serde::Serialize,
+    std::time::Duration,
+    tokio::sync::mpsc,
+};
+
+pub struct ClickhouseInserter<T> {
+    pub sender: mpsc::Sender<T>,
+}
+
+impl<T: Row + Serialize + Send + Sync + 'static> ClickhouseInserter<T> {
+    async fn run(client: clickhouse::Client, table_name: String, mut rx: mpsc::Receiver<T>) {
+        match client.inserter(&table_name) {
+            Ok(inserter) => {
+                let mut inserter = inserter
+                    .with_period(Some(Duration::from_secs(1)))
+                    .with_max_rows(100)
```

Comment:
maybe these should be constants defined above? also can you leave some comments to make these params more understandable?

Reply:
in particular some comments to explain how the batching exactly works, since it isn't obvious from these params